Pipes - <Nkiru> - <Calculator.rb>

```diff
@@ -0,0 +1,103 @@
+# Nkiru Onwuneme
+# Last edited 8/11/17
+# Create a calculator
+
+##Issues:  user entry set to interger or in Method - fixed
+# consolidate methods
+
+# Methods for computing math operations
+def add(num_1, num_2)
+  result = num_1 + num_2
+  return result
+end
+
+def subtract(num_1, num_2)
+  result = num_1 - num_2
+  return result
+end
+
+def multiply(num_1, num_2)
+  result = num_1 * num_2
+  return result
+end
+
+# method to check if denominator is o
+def divide(num_1, num_2)
+  if num_2 == 0
+    puts "Invalid number, numerator indivisible by 0"
+  else
+    result = num_1.to_f/num_2.to_f
+  end
+  return result.round(6)
+end
+
+def exponent(num_1, num_2)
+  result = num_1 ** num_2
+  return result
+end
+
+def modulo(num_1, num_2)
+    result = num_1 % num_2
+    return result
+end
+
+def num_check(num)
+  begin
+   num = gets.chomp
+   if num.include?(".")
+    return num = Float(num)
+    else
+  return num = Integer(num)
```

Comment:
Watch your spacing here - the `else` should be at the same level as the `if` above, and the `return` on line 50 should be indented more. When in doubt, use Atom's auto-indent.